magento/pwa-studio
# PWA-707: Update Venia's accordions to match the style guides accordions

## Description

Update accordions throughout the application to match those of the style guide.

## Related Issue
https://jira.corp.magento.com/browse/PWA-707

## Acceptance 



### Verification Stakeholders
@jimbo 
@dpatil-magento 
@anthoula 

### Specification



### Verification Steps
1. Navigate to cart and checkout and verify the accordions meet the style guide
2. Verify all instances of the style guide are working as expected
3. Verify instances of black links have been updated to blue links (such as Sign in on cart)

## Screenshots / Screen Captures (if appropriate)

## Checklist
